packages/v4/patternfly-docs/content/design-guidelines/components/banner/banner.md

```diff
@@ -21,23 +21,42 @@ PatternFly offers 5 different banner types detailed below.
 |Warning |Orange <br />(--pf-global--warning-color--100) | Black | Use for warning or medium severity messages |
 <br />
 PatternFly suggests that users adopt one of these 5 colors, as they’ve been tested against their text colors for accessibility. However, if colors outside of these suggestions are used, we encourage using discretion when selecting the inner text color.
+
+### When to use
  
-### Placement
+Banners are flexible and can be used for any content and for several use cases outlined in the following examples.
  
-All banners should be a solid bar that spans the top and/or bottom of either:
-1. The full page, above the masthead.
+#### To indicate that an application/site is offline
```

Comment:
Since these aren't making it in the ToC, I wonder if we should just have them as a bulleted list? I think Sylvana has being doing that for other guidelines, and just indenting the image to be right below the bullet